Sming/Makefile-project.mk

```diff
@@ -72,6 +72,17 @@ else
       ESP_HOME ?= /opt/esp-open-sdk
       include $(SMING_HOME)/Makefile-linux.mk     
   endif
+  ifeq ($(UNAME),FreeBSD)
+      # Freebsd Detected
+      UNAME := FreeBSD
+
+      # Default SMING_HOME. Can be overriden.
+      SMING_HOME ?= /usr/home/shurik/src/esp8266/Sming/Sming
```

Comment:
06 июня 2015 г. 2:56 пользователь "Skurydin Alexey" <
notifications@github.com> написал:
>
> > +  ifeq ($(UNAME),FreeBSD)
> > +      # Freebsd Detected
> > +      UNAME := FreeBSD2
> > +
> > +      # Default SMING_HOME. Can be overriden.
> > +      SMING_HOME ?= /usr/home/shurik/src/esp8266/Sming/Sming
>
> It is absolute path in your OS? Can it be replaced by relative?
>
Yes, it can replaced by relative path like /usr/local/esp-open-sdk or more
relative ~/esp-open-sdk or similar path.
Or I miss you question??